styles.css

```diff
@@ -0,0 +1,349 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Catamaran', sans-serif;
+}
+
+body {
+    background-image: url('img/guitar-blur.jpeg');
+    background-position: center;
+    background-attachment: fixed;
+    top:0;
+    font-size: 62.5%;
+    text-align: center;
+    color: rgb(240, 240, 210);
+}
+
+html {
+    scroll-behavior: smooth;
+}
+/* add conditional CSS for different*/
```

Comment:
Wouldn't leave comments detailing a todo item in code submitted for pull request

Sounds like this needs to be completed before merging to master